modules/registry-configuring-storage-aws-user-infra.adoc

```diff
@@ -36,3 +36,11 @@ storage:
     bucket: <bucket-name>
     region: <region-name>
 ----
+
+[WARNING]
+====
```

Comment:
I would suggest removing the link to the blog, and simplifying the warning as follows: 

[WARNING]
====
To secure your registry images in AWS, link:https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-publicaccessblockconfiguration.html[block public access]
to the S3 bucket. 
====

Reply:
Done.